actions/ci-benchmarking/action.yml

```diff
@@ -0,0 +1,126 @@
+name: 'Go Benchmark Action'
+description: 'Run Go benchmarks and optionally publish results to GitHub Pages'
+inputs:
+  benchmarks-pr:
+    description: 'Comma-separated list of benchmarks to run on PRs (e.g., BenchmarkOne, BenchmarkTwo). Set to "from-pr" to extract from PR description.'
+    required: false
+    default: 'from-pr'
+
+  benchmarks-merge:
+    description: 'Comma-separated list of benchmarks to run on merges (e.g., BenchmarkOne, BenchmarkTwo).'
+    required: false
+    default: ''
```

Comment:
I see in the readme that there is a default set to `BenchmarkAdd`?

Reply:
That's right - it's a default benchmark to show the functionality of the action to consumers and to detect regressions/irregularities in the action behavior ("who is watching the watcher?")